src/pages/graphql/schema/checkout/queries/get-payment-config.md

```diff
@@ -0,0 +1,251 @@
+---
+title: getPaymentConfig query
+---
+
+# getPaymentConfig query
+
+The `getPaymentConfig` query returns the payment configuration for the available payment methods for a specific `PaymentLocation`:
+
+* CHECKOUT
+* MINICART
+* CART
+* PRODUCT_DETAIL
+* ADMIN
+
+These payment methods for the `getPaymentConfig` query are the ones supported in [Payment Services](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/payments-checkout/payments-options.html):
+
+* PayPal Hosted Fields
+* Apple Pay
+* PayPal Smart Buttons
+
+## getPaymentConfig object
+
+The `getPaymentConfig` object will differ on which attributes must contain depending on the payment method.
```

Comment:
```suggestion
The `PaymentConfigOutput` contains details about each configured payment method.
```

Reply:
I can't correct the output information from the GH PR review boxes. We have to have a meeting where I show you how to assemble this info.